modules/core/include/opencv2/core/version.hpp

```diff
@@ -53,7 +53,7 @@
 #define CV_VERSION_MAJOR    3
 #define CV_VERSION_MINOR    4
 #define CV_VERSION_REVISION 0
-#define CV_VERSION_STATUS   ""
+#define CV_VERSION_STATUS   "-dev"
```

Comment:
Commit https://github.com/hunter-packages/opencv/pull/21/commits/047764f476e429e0ff63f9e23f7ac96f59539356 doesn't belong to 3.4.0

Same commit upstream only in master:
* https://github.com/opencv/opencv/commit/047764f476e429e0ff63f9e23f7ac96f59539356

3.4.0 tag:
* https://github.com/opencv/opencv/commit/6d4f66472e14b29b8e1623859cfebfdc67f677c3

Reply:
Whoops. I rebased and that one just deleted itself. Thanks.